idea/resources/inspectionDescriptions/UnnecessaryJavaUsage.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+Reports unnecessary java usage.
```

Comment:
This is not clear. A better description would be "Use of Java API that has a Kotlin equivalent" or something like that.